packages/babel-polyfill/package.json

```diff
@@ -12,6 +12,6 @@
   "main": "lib/index.js",
   "dependencies": {
     "core-js": "^2.6.5",
-    "regenerator-runtime": "^0.12.0"
+    "regenerator-runtime": "^0.13.2"
```

Comment:
I'd like to do this in #9442 to give credits for this update to @MattiasBuelens.
`^0.12.0` still loads the old regenerator version anyway, which added `regeneratorRuntime` to the global object.

Reply:
I don't care. Anyway, it's also required here since `@babel/runtime-corejs3` should be updated too.

Reply:
If it is required here, I would do it here and give credit in changelog.